app/src/app/modules/main-page/components/profile/profile.component.html

```diff
@@ -0,0 +1,52 @@
+<majk-navigation></majk-navigation>
+
+<div>
+  <button class="button is-white"><a [routerLink]="['/home']">
+      <h1 style="color:black; font-size:30px">&larr;</h1>
+    </a>
+  </button>
+</div>
```

Comment:
I to też by było niepotrzebne bo z profilu można się gdzie indziej dostać przez navbar